doc/specs/Authentication.md

```diff
@@ -0,0 +1,473 @@
+# Authentication system
+
+## Metadata
+
+Author: DutchDude
+
+### Changelog
+
+**2014-12-31**
+
+* Added second half of use cases
+* Added a bunch of business rules
+
+**2014-12-24**
+
+* Initial draft with first half
+
+## Table of contents
+
+1. Motivation
+1. Business rules
+1. Use cases
+1. Data model
+
+## Motivation
+
+What.CD is a walled garden, in which its community shares lots of great content.
+To keep the community in great shape and to ensure quality control, the website
+is not publicly accessible. Restricted access ensures that bad people are (mostly)
+kept out and keeps the site community and economy healthy.
+
+## Business rules
+
+Business rules are principles that should be honored throughout the application.
+They are not technical requirements, but specify what is needed to "help the
+organization achieve its goals." (Wikipedia)
+
+* Users need always be identifiable
```

Comment:
Not going to fix this, I'm not going for a prize here :stuck_out_tongue_winking_eye: